test/RiskManagerV1.test.js

```diff
@@ -331,31 +328,33 @@ describe("RiskManagerV1", () => {
         await notifyLiquidation()
 
         auctionAddress = await riskManagerV1.depositToAuction(
-          mockIDeposit.address
+          depositStub.address
         )
 
         // Simulate that someone takes a partial offer on the auction.
         await tbtcToken.mint(bidder.address, auctionLotSize)
         await tbtcToken.connect(bidder).approve(auctionAddress, auctionLotSize)
         auction = new ethers.Contract(auctionAddress, Auction.abi, owner)
-        auction.connect(bidder).takeOffer(to1ePrecision(25, 16))
+        await auction.connect(bidder).takeOffer(to1ePrecision(25, 16))
 
         // Simulate that deposit was liquidated by someone else.
-        await mockIDeposit.mock.currentState.returns(depositLiquidatedState)
+        await tbtcToken.mint(owner.address, auctionLotSize)
+        await tbtcToken.approve(depositStub.address, auctionLotSize)
```

Comment:
Can we do `tbtcToken.connect(owner).approve(...)` and `depositStub.connect(owner).purchaseSignerBondsAtAuction` for clarity? I was not sure at first why we are minting tokens to owner's account.

Reply:
https://github.com/keep-network/coverage-pools/pull/79/commits/17844aa27212eb132b5743a7088459cca6fa4217